OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -47,6 +51,8 @@ class Parachutable : INotifyParachuteLanded
 		readonly ParachutableInfo info;
 		readonly IPositionable positionable;
 
+		bool onWater;
```

Comment:
abcdefg30 is right here, this doesn't need to be global. Same in the other file below.